en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -37,7 +34,7 @@ To start multiple instances (on separate ports):
 
 Ground stations such as *QGroundControl* connect to all instances using the normal UDP port 14550 (all traffic goes to the same port).
 
-Developer APIs such as *MAVSDK* or *MAVROS* connect on the UDP port 14540 (first instance), UDP port 14541 (second instance), and so on.
+Developer APIs such as *MAVSDK* or *MAVROS* connect on the UDP port 14540 (first instance), UDP port 14541 (second instance), and so on. 
```

Comment:
Pending discussion https://github.com/PX4/PX4-user_guide/issues/1035#issuecomment-777095536 - I think that once you hit 14549 all instances need to connect to this same port.